cpp/test/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cxx_add_compile_options(MSVC /wd4310)
 # Core unit tests don't build under VS2013 without this.
 cxx_add_compile_options(MSVC /EHa)
 
-if (NOT BOND_SKIP_CORE_TESTS)
+if (NOT BOND_SKIP_COMPAT_TESTS)
```

Comment:
Also consider updating AppVeyor and Travis configs to include the new switch where appropriate.